fixed_bias: false
fixed_calibr: false
gyro_noise: 0.005
acc_noise: 0.05
gyro_bias_noise: 0.001
acc_bias_noise: 0.01
noise_camera_q: 0.1